src/sentry.ts

```diff
@@ -29,10 +29,10 @@ export const extendSentry = (logger: Cosmas, options: { sentry: string | true; s
         sentry.init({ dsn: options.sentry });
     }
 
-    const originalWrite = logger[pino.symbols.streamSym].write;
+    const originalWrite = (logger as any)[pino.symbols.streamSym].write;
```

Comment:
Why do we still need the any cast? Not public API even after update?

Reply:
Still not. I will fix that in the future MR